### ISSUE TITLE
Mis-named file in 7.711 release

### ISSUE THREAD

**Issue by pghmcfc** [elastic/elasticsearch-perl#202]
The file `lib/Search/Elasticsearch/Client/7_0/Direct/SearchableSnapshots copy.pm` should be called `lib/Search/Elasticsearch/Client/7_0/Direct/SearchableSnapshots.pm`; as it stands, attempts to build the module result in this error:

```
$ cd Search-Elasticsearch-7.711
$ perl Makefile.PL INSTALLDIRS=vendor NO_PACKLIST=1 NO_PERLLOCAL=1
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Search::Elasticsearch
$ make -j4
make: *** No rule to make target 'lib/Search/Elasticsearch/Client/7_0/Direct/SearchableSnapshots', needed by 'manifypods'.  Stop.
make: *** Waiting for unfinished jobs....
```
As a workaround I have renamed the file and changed the references to it in `t/author-no-tabs.t`, `t/author-eol.t` and `MANIFEST`, which is enough to get the module to build and pass its test suite.

**Comment by gyarakilaszlo** [comment]
I can confirm this issue, and pghmcfc's workaround is working well.

**Comment by ezimuel** [comment]
Thanks for the alert, this is embarrassing :disappointed_relieved:. I'll fix this with the path release `7.11.1`.

**Comment by ezimuel** [comment]
Fixed with [7.11.1](https://github.com/elastic/elasticsearch-perl/releases/tag/version_7.711001) patch release.
